docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -1238,72 +1283,70 @@ quarkus tls lets-encrypt issue-certificate \
 ----
 <1> Set your domain name.
 <2> Provide your contact email address that Let's Encrypt can use to contact you in case of any issues with your Let's Encrypt account.
-<3> Set your application management URL, which you can use to handle ACME challenges. 
-Use `https://localhost:8443/` if you choose not to enable a management router in the <<lets-encrypt-prerequisites>> step.
+<3> Set your application management URL, which you can use to handle ACME challenges.
+Use `https://localhost:8443/` if you choose not to enable a management router in the <<lets-encrypt-prerequisites,Let's Encrypt prerequisites>> step.
 
-During the processing of the `issue-certificate` command, the TLS registry CLI performs the following tasks:
+.. During the processing of the `issue-certificate` command, the TLS registry CLI performs the following tasks:
 
 * Checks if the application is prepared to serve the challenge.
 * Creates and records Let's Encrypt account information.
 * Issues a Let's Encrypt certificate request.
 * Interacts with the Quarkus application to resolve ACME challenges.
-
-Once the Let's Encrypt certificate chain and private key have been successfully acquired, they are converted to PEM format and copied to your application's `.letsencrypt` folder.
++
+NOTE: Once the Let's Encrypt certificate chain and private key have been successfully acquired, they are converted to PEM format and copied to your application's `.letsencrypt` folder.
 The TLS registry is informed that a new certificate and private key are ready and reloads them automatically.
-
-Now, access your application's endpoint using `https://your-domain-name:8443/` again.
++
+. Access your application's endpoint using `https://your-domain-name:8443/` again.
 Confirm in the browser that the Let's Encrypt certificate authority is now signing your domain certificate.
-
++
 Note that currently, the `issue-certificate` command implicitly creates a Let's Encrypt account to make it easy for users to get started with the ACME protocol.
 Support for the Let's Encrypt account management will evolve further.
 
-[[lets-encrypt-renew-certificate]]
-=== Renew certificate
-
-Renewing certificates is similar to issuing the first certificate, but it requires an existing account created during the <<lets-encrypt-issue-certificate>> step.
-
-Run the following command to renew your Let's Encrypt certificate:
+. Renew certificate:
```

Comment:
```suggestion
. Renew a certificate:
```
Should it be a step in the same procedure, though? I'm guessing you don't need to renew a certificate that you just issued, no?

Reply:
@cescoffier Clement, is this about applying the newly issued certificate and replacing the previous one?

Reply:
Or maybe this is a separate section since back then, it was sorted by headings, not by numbered steps as I tried to apply, thus it was not meant to be a part of the same procedure.